MuonAnalysis/MuonAssociators/src/PropagateToMuonCore.cc

```diff
@@ -86,7 +60,9 @@ void PropagateToMuon::init(const edm::EventSetup &iSetup) {
     barrelHalfLength_ = endcapDiskPos_[2]->position().z();
 }
 
-FreeTrajectoryState PropagateToMuon::startingState(const reco::Candidate &reco) const {
+PropagateToMuonCore::~PropagateToMuonCore() {}
```

Comment:
```suggestion
```

Reply:
Can't be removed because of the way this class is used in `L1MuonMatcherAlgo`.

Reply:
Indeed not, but the empty body {} could be moved inside the class definition just for simplicity (suggested somewhere above).

Reply:
Done.